app/View/Label.php

```diff
@@ -125,6 +125,7 @@ public function render(callable $callback = null)
                     if (($barcode2DType != 'none') && (!is_null($barcode2DType))) {
                         switch ($settings->label2_2d_target) {
                             case 'ht_tag': $barcode2DTarget = route('ht/assetTag', $asset->asset_tag); break;
+                            case 'asset_tag' : $barcode2DTarget = basename(route('ht/assetTag', $asset->asset_tag)); break;
```

Comment:
I'm not sure what this is doing - can you clarify?

Reply:
so ht_tag returns a url of `blah.com/ht/{asset_tag}`, and `basename()` returns the trailing name of the path. without writing another method or route we get the value we want here.

Reply:
im not sure if this is bad practice, but if I should write a getter for it I will 👍

Reply:
> so ht_tag returns a url of blah.com/ht/{asset_tag}, and basename() returns the trailing name of the path. without writing another method or route we get the value we want here.

I feel stupid for asking this, but if the info should just be an asset tag, wouldn't this work?

```php
case 'asset_tag' : $barcode2DTarget = $asset->asset_tag; break;
```